src/main/java/com/school/project/controller/UserController.java

```diff
@@ -1,9 +1,51 @@
 package com.school.project.controller;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.school.project.dto.UserDto;
+import com.school.project.repository.UserRepository;
+import com.school.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
+@RequestMapping
 public class UserController {
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserRepository userRepository;
+
+    @PostMapping("/user/creation")
+    public void createUser(@RequestBody UserDto userDto) {
+        userService.create(userDto);
+    }
+
+    @GetMapping("/user/{id}")
```

Comment:
It is better to name the URL "/users", so , it denotes to a collections of users. The the GET url looks natural: "/users/25" - give me the user number 25.